include/mbedtls/sha512.h

```diff
@@ -33,13 +33,6 @@
 #include <stddef.h>
 #include <stdint.h>
 
-#define MBEDTLS_ERR_SHA512_HW_ACCEL_FAILED                -0x0039  /**< SHA-512 hardware accelerator failed */
-
-#if ( defined(__ARMCC_VERSION) || defined(_MSC_VER) ) && \
-    !defined(inline) && !defined(__cplusplus)
-#define inline __inline
-#endif
-
```

Comment:
Why has this been removed?

Reply:
Changes made on file prior to another commit made yesterday. I have merged the changes into my version, reviewed the deprecated minimally and reloaded file. Apologies.